packages/_private/eslint-config-syn/package.json

```diff
@@ -39,7 +39,8 @@
     "directory": "packages/eslint-config-syn"
   },
   "scripts": {
-    "lint:js": "eslint ."
+    "lint:js": "eslint .",
+    "format.eslint": "eslint . --fix"
```

Comment:
In general naming feels not really consistent. Not really a show stepper and maybe it makes more sense to rethink it when we move formatting completly to prettier, because then we could just call it:

`format` and `lint` everywhere.

Reply:
So I think improving the script names everywhere and bringing more consistency could definitely help, but I'm fine with approaching that in the other ticket for using prettier everywhere for formatting.